ci/travis/install-osx.sh

```diff
@@ -32,7 +32,7 @@ python --version
 
 # Build NuPIC
 cd ${TRAVIS_BUILD_DIR}
-python setup.py install --user
+ARCHFLAGS="-arch x86_64" python setup.py install --user
```

Comment:
If this is still required for OS X builds, we'll need to add this to the README until we figure out how automate it.